lib/cursor.js

```diff
@@ -17,25 +17,25 @@ module.exports = class Cursor extends Readable {
       Cursor.prototype[operation] = (data) => {
         this._options[operation] = data;
         return this;
-      }
+      };
     });
   }
 
   map(fn) {
-    const result = []
+    const result = [];
 
     return new Promise((resolve) => {
       const loop = () => {
         this.next()
-          .then(doc => {
-            if (!doc) { return resolve(result); }
+          .then((doc) => {
+            if (!doc) { return void resolve(result); }
```

Comment:
Here's a non-automated change to ensure we always return `undefined` from from the function. `resolve(x)` already results in undefined, so this just makes the implicit behavior explicit.